### DISCUSSION TITLE
Starting out with zotero-sync and Bookends

### DISCUSSION THREAD

**Discussion by joshuawagner** [retorquere/zotero-sync#15] [Q&A] [answered]
Hi, I previously used https://github.com/cboulanger/bookends-api. That is now deprecated and points to this project and the [Bookends library](https://github.com/cboulanger/zotero-sync-bookends) that he made for **zotero-sync**. 
I followed the instructions on his page to install the library but I can't figure out what to do with **zotero-sync** to make it all work. Do you mind walking me through it, please?

**Comment by cboulanger** [comment, marked as answer]
Instructions are here: https://github.com/cboulanger/zotero-sync-bookends

This repo contains code on which the bookends sync depends, it doesn't do anything on its own.

**Reply by joshuawagner** [reply]
Alright. That makes sense.

Can you explain what I need to do with the environment variables?

**Reply by cboulanger** [reply]
I am happy to answer to an issue in https://github.com/cboulanger/zotero-sync-bookends/issues . I'd suggest you close this issue since the question has nothing to do with retorquere/zotero-sync - ok?
